Products/CMFPlone/skins/plone_templates/standard_error_message.py

```diff
@@ -27,6 +27,10 @@
 error_traceback = kwargs.get('error_traceback', None)
 error_value = kwargs.get('error_value', None)
 
+if "text/html" not in context.REQUEST.getHeader('Accept', ''):
+    context.REQUEST.RESPONSE.setHeader("Content-Type", "application/json")
+    return '{"error_type": "%s"}' % error_type
+
```

Comment:
As a developer I would really love to have the traceback and other error values available. But OTOH it make the response large and we want small responses in production. Also we would need to take the permissions into account. 
What about adding the additional information only if zope is running in debug mode (foreground)?